src/routes/Projects/routes/Project/containers/TreeView/TreeView.js

```diff
@@ -11,7 +11,7 @@ import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
 import { actions as TabActions } from '../../modules/tabs'
 import { devshare, helpers } from 'redux-devshare'
-const { isLoaded, isEmpty, dataToJS } = helpers
+const { isLoaded, isEmpty, dataToJS, toJS } = helpers
```

Comment:
Yes dude! Look at you! 🛩 

If we wanted to use `react-redux-firebase` (like on the v0.3.0 branch) we can import these from its top level (it is a little ahead of `redux-devshare`):

```js
import { isLoaded, isEmpty, dataToJS, toJS } from 'react-redux-firebase'
```

Lets not worry about that for now, you can leave it how it is.